accord-core/src/main/java/accord/local/PreLoadContext.java

```diff
@@ -91,15 +93,16 @@ default void forEachId(Consumer<TxnId> consumer)
      */
     default Unseekables<?> keys() { return RoutingKeys.EMPTY; }
 
-    default KeyHistory keyHistory() { return NONE; }
+    default LoadKeys loadKeys() { return NONE; }
+    default LoadKeysFor loadKeysFor() { return WRITE; }
```

Comment:
Should we have a default here even? Could be surprising for an implementer, easy to overlook. Having to implement it at least would make a person think / consider.

Reply:
I don't know. I am inclined to keep it as is, because we have a LOT of these classes and most don't need to load key information. But I don't feel incredibly strongly about it.